### PR TITLE
Update swift-certificates to 1.0.1, swift-crypto to 3.0.0

### PR DESCRIPTION
**Pull request by yim-lee**
rdar://113969026
rdar://115794818

**This PR requires update-checkout changes**

### CODE REVIEW

**Comment by yim-lee** [comment]
@swift-ci please smoke test

**Comment by yim-lee** [comment]
@swift-ci please test

**Comment by MaxDesiatov** [comment]
I guess 1.0.0 is unsuitable, since https://github.com/apple/swift-certificates/pull/142 is not included in that release?

**Comment by dnadoba** [comment]
[We have just released `1.0.1` with the fix for windows.](https://github.com/apple/swift-certificates/releases/tag/1.0.1)

**Comment by MaxDesiatov** [comment]
@swift-ci test

File: Package.swift [hunk 1]
```diff
@@ -732,10 +732,10 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // dependency version changes here with those projects.
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.2.2")),
         .package(url: "https://github.com/apple/swift-driver.git", branch: relatedDependenciesBranch),
-        .package(url: "https://github.com/apple/swift-crypto.git", .upToNextMinor(from: "2.5.0")),
+        .package(url: "https://github.com/apple/swift-crypto.git", .upToNextMinor(from: "3.0.0")),
         .package(url: "https://github.com/apple/swift-system.git", .upToNextMinor(from: "1.1.1")),
         .package(url: "https://github.com/apple/swift-collections.git", .upToNextMinor(from: "1.0.1")),
-        .package(url: "https://github.com/apple/swift-certificates.git", .upToNextMinor(from: "0.6.0")),
+        .package(url: "https://github.com/apple/swift-certificates.git", .upToNextMinor(from: "1.0.0")),
```
Comment (MaxDesiatov) [inline Package.swift:738]:
```suggestion
        .package(url: "https://github.com/apple/swift-certificates.git", .upToNextMinor(from: "1.0.1")),
```

**Comment by MaxDesiatov** [comment]
@swift-ci test

**Comment by yim-lee** [comment]
Thanks @dnadoba @MaxDesiatov. I will make the update-checkout changes to point to swift-certificates 1.0.1.

**Comment by yim-lee** [comment]
https://github.com/apple/swift/pull/68768

@swift-ci please test

**Review by MaxDesiatov — APPROVED** [review]
Thanks!

**Review by Sambit650 — APPROVED** [review]

**Pull request merged**